src/libcore/ops.rs

```diff
@@ -445,7 +445,10 @@ rem_impl! { usize u8 u16 u32 u64 isize i8 i16 i32 i64 }
 impl Rem for f32 {
     type Output = f32;
 
-    // see notes in `core::f32::Float::floor`
+    // The builtin f32 frem doesn't compile when targeting
+    // msvc; use f64 rem instead.
+    // FIXME: This is a bug; trans should be fixed instead
+    // of working around it here.
```

Comment:
Perhaps this comment could stay referencing `Float::floor`? This is basically the same fix where the header files for MSVC has `fmodf` as just a wrapper around `fmod`, so the comment over there is the same for here. It's also not clear that trans should be fixed so much as LLVM itself.